### PR TITLE
fix(cta): card link and other story fixes

### PR DESCRIPTION
**Pull request by NSDrowned**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5541

### Description

- React: Aligned to the left and added same padding as web components
- Web Component: Added missing corner icon

### CODE REVIEW

File: packages/react/src/components/CTA/__stories__/cta.stories.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.bx--card {
+  margin-top: 1rem;
+  margin-left: -1rem;
+}
```
Comment (kennylam) [inline packages/react/src/components/CTA/__stories__/cta.stories.scss:11]: I don't think these are needed? Removing them makes the story layout identical to the web components version.

**Review by kennylam — COMMENTED** [review]
Thanks for this @NSDrowned! Just a small observation

File: packages/react/src/components/CTA/__stories__/cta.stories.scss (hunk 1 above)
Comment (NSDrowned) [reply packages/react/src/components/CTA/__stories__/cta.stories.scss:11]: You're right 🤔 Weird... my fork was bit behind when I worked on it. Perhaps that's why I added them cause I see no reason now lol. Thanks!

**Review by shixiedesign — APPROVED** [review]
Visually unchanged. Goal is to make sure`CTA - card` uses `Card link`. Please get additional dev approval!

**Pull request merged**